### PR TITLE
Docs 3447 update/create advanced system setting articles

### PR DESCRIPTION
**Pull request by micjohnson777**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/SystemSettings/Advanced/ManageCronJobsSCALE.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+{{< include file="/_includes/AdvancedSettingsWarningSCALE.md" type="page" >}}
+
+The **Cron Jobs** widget on the **System > Advanced** screen displays **No Cron Jobs configured** until you add a cron job, and then it displays information on cron job(s) configured on the system.
+
+![AdvancedSettingsCronJobWidget](/images/SCALE/22.02/AdvancedSettingsCronJobWidget.png "SCALE Advanced Settings Cron Job Widget") 
+
+Click **Add** to open the **Add Cron Job** configuration sceen to create a new cron job, or if you want to modify an existing job, click anywhere on the item to open the **Edit Cron Jobs** configuration screen populated with the settings for that cron job. 
+The **Add Cron Job** and **Edit Cron Job** configuration screens display the same settings. 
+
+![ConsoleConfigScreen](/images/SCALE/22.02/ConsoleConfigScreen.png "SCALE Console Settings Screen") change
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/SystemSettings/Advanced/ManageCronJobsSCALE.md:23]:
```suggestion
![ConsoleConfigScreen](/images/SCALE/22.02/ConsoleConfigScreen.png "SCALE Console Settings Screen")
```

File: content/SCALE/SCALETutorials/SystemSettings/Advanced/ManageSEDsSCALE.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: "Managing SEDs"
+description: "This article provides information on adding or modifying self-encrypting drive (SED) user and global passwords inSCALE."
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/SystemSettings/Advanced/ManageSEDsSCALE.md:3]:
```suggestion
description: "This article provides information on adding or modifying self-encrypting drive (SED) user and global passwords in SCALE."
```

File: content/SCALE/SCALETutorials/SystemSettings/Advanced/ManageSEDsSCALE.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+The **Self-Encrypting Drive(s)** widget on the **System > Advanced** screen allows you set the user and global SED password in SCALE.
+
+## Managing Self-Encrypting Drives
+
+The **Self-Encrypting Drive** (SED) widget displays the ATA security user and password configured on the system. 
+
+![AdvancedSystemSEDWidget](/images/SCALE/22.02/AdvancedSystemSEDWidget.png "SCALE Advanced Settings Self-Encrypting Drive Widget") 
+
+Click **Configure** to open the **Self-Encrypting Drive)** configuration screen.
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/SystemSettings/Advanced/ManageSEDsSCALE.md:21]:
```suggestion
Click **Configure** to open the **Self-Encrypting Drive** configuration screen. 
```

File: content/SCALE/SCALETutorials/SystemSettings/Advanced/ManageSEDsSCALE.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+The **Self-Encrypting Drive** (SED) widget displays the ATA security user and password configured on the system. 
+
+![AdvancedSystemSEDWidget](/images/SCALE/22.02/AdvancedSystemSEDWidget.png "SCALE Advanced Settings Self-Encrypting Drive Widget") 
+
+Click **Configure** to open the **Self-Encrypting Drive)** configuration screen. 
+The **Self-Encrypting Drive** configuration screen allows users set the ATA security user and create a SED global password.
+
+![AdvancedSystemSelfEncryptingDriveConfigScreen](/images/SCALE/22.02/AdvancedSystemSelfEncryptingDriveConfigScreen.png "SCALE Advanced Settings Self-Encrypting Drive screen") 
+
+Select the user passed to camcontrol security -u to unlock SEDs from the **ATA Security User** dropdoan list. Options are **USER** or **MASTER**.
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/SystemSettings/Advanced/ManageSEDsSCALE.md:26]:
```suggestion
Select the user passed to *camcontrol security -u* to unlock SEDs from the **ATA Security User** dropdoan list. Options are **USER** or **MASTER**.
```

File: content/SCALE/SCALETutorials/SystemSettings/Advanced/ManageSysLogsSCALE.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+{{< /hint >}}
+
+Click **Configure** to open the **Syslog** configuration screen. 
+The **Syslog** configuration screen settings specify the logging level the system uses to record system events, the syslog server DNS host name or IP, the transport protocol it uses, and if using TLS, the certificate and certificate authority (CA) for that server, and finally if it uses the system dataset to store the logs.
+
+![SyslogConfigScreen](/images/SCALE/22.02/SyslogConfigScreen.png "SCALE Syslog Settings Screen") 
+
+Enter the remote syslog server DNS host name or IP address in **Syslog Server**. To use non-standard port numbers like *mysyslogserver:1928*, add a colon and the port number to the host name. Log entries are written to local logs and sent to the remote syslog server.
+
+Enter the [transport protocol](https://tools.ietf.org/html/rfc8095) for the remote system log server connection in **Syslog Transport**. Selecing Transport Layer Security (TLS) displays the **Syslog TLS Certificate** and **Syslog TSL Certificate Authority** fields.
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/SystemSettings/Advanced/ManageSysLogsSCALE.md:30]:
```suggestion
Enter the [transport protocol](https://tools.ietf.org/html/rfc8095) for the remote system log server connection in **Syslog Transport**. Selecting Transport Layer Security (TLS) displays the **Syslog TLS Certificate** and **Syslog TSL Certificate Authority** fields. 
```

File: content/SCALE/SCALEUIReference/SystemSettings/AdvancedSettingsScreen.md [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+| **Enabled** | Select to enable this cron job. Leave cleared to disable the cron job without deleting it. |
+{{< /expand >}}
+
+## Init/Shutdown Scripts Widget
+The **Init/Shutdown Scripts** widget displays **No Init/Shutdown Scripts configured** until you add either a command or script, then the widget lists the scrips configured on the system.
+
+![AdvancedSystemInitShutdownScriptWidget](/images/SCALE/22.02/AdvancedSystemInitShutdownScriptWidget.png "SCALE Advanced Settings Init/Shutdown Script Widget") 
+
+**Add** opens the **[Add Init/Shutdown Script](#add-or-edit-initshutdown-script-configuration-screens)** configuration screen.
+Any script listed is a link that opens the **[Edit Inti/Shutdown Script](#add-or-edit-initshutdown-script-configuration-screens)** configuration screen populated with the settings for that script.
```
Comment (clairewrtr) [inline content/SCALE/SCALEUIReference/SystemSettings/AdvancedSettingsScreen.md:105]:
```suggestion
Any script listed is a link that opens the **[Edit Init/Shutdown Script](#add-or-edit-initshutdown-script-configuration-screens)** configuration screen populated with the settings for that script.
```

File: content/SCALE/SCALEUIReference/SystemSettings/AdvancedSettingsScreen.md [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+![AddInitShutdownScriptConfigScreen](/images/SCALE/22.02/AddInitShutdownScriptConfigScreen.png "SCALE Init/Shutdown Script Settings Screen") 
+
+| Settings | Description |
+|----------|-------------|
+| **Description** | Comments about this script. |
+| **Type** | Select Command for an executable or Script for an executable script. |
+| **Command** | Enter the command with any options. |
+| **Script** | Select the script. The script runs using [dash(1)](https://manpages.debian.org/testing/dash/sh.1.en.html "dash(1) Page"). |
+| **When** | Select when the command or script runs from the dropdown list. Options are **Pre Init** for early in the boot process, after mounting file systems and starting networking. **Post Init** runs at the end of the boot process, before Linux services start. **Shutdown** runs during the system power-off process. |
+| **Enabled** | Select to enable this script. When left cleared, it disable the script without deleting it. |
```
Comment (clairewrtr) [inline content/SCALE/SCALEUIReference/SystemSettings/AdvancedSettingsScreen.md:120]:
```suggestion
| **Enabled** | Select to enable this script. When left cleared, it disables the script without deleting it. |
```

File: content/SCALE/SCALEUIReference/SystemSettings/AdvancedSettingsScreen.md [hunk 8]
```diff
@@ -0,0 +1,209 @@
...
+### Add or Edit Sysctl Configuration Screen
+The **Add Sysctl** or **Edit Sysctl** configuration screen settings lets users set up tunables that configure kernel parameters at runtime.
+{{< expand "Click Here for More Information" "v" >}}
+
+![AddSysctlConfigScreens](/images/SCALE/22.02/AddSysctlConfigScreen.png "SCALE Add Sysctl Screen") 
+
+| Settings | Description |
+|----------|-------------|
+| **Variable** | Enter the name of the sysctl variable to configure. sysctl tunables are used to configure kernel parameters while the system is running and generally take effect immediately. |
+| **Value** | Enter a [sysctl](https://man7.org/linux/man-pages/man8/sysctl.8.html) value to use for the loader, **sysctl**, or **rc.conf** variable. |
```
Comment (clairewrtr) [inline content/SCALE/SCALEUIReference/SystemSettings/AdvancedSettingsScreen.md:140]:
```suggestion
| **Value** | Enter a [sysctl](https://man7.org/linux/man-pages/man8/sysctl.8.html) value to use for the **loader**, **sysctl**, or **rc.conf** variable. |
```

File: content/SCALE/SCALEUIReference/SystemSettings/AdvancedSettingsScreen.md [hunk 9]
```diff
@@ -0,0 +1,209 @@
...
+The **Isolate GPU PCI's ID** configuration screen allows you to add GPU devices to your system. 
+{{< expand "Click Here for More Information" "v" >}}
+GPU passthrough allows the TrueNAS SCALE kernel to directly present an internal PCI GPU to a virtual machine (VM).
+
+![IsolatedGPUPCIIDsConfigScreen](/images/SCALE/22.02/IsolatedGPUPCIIDsConfigScreen.png "SCALE Advanced Settings Isolated GPU PCI ID Screen") 
+
+The GPU device acts like the VM is driving it, and the VM detects the GPU as if it is physically connected. Select the GPU device ID from the dropdown list. 
+To isolate a GPU you must have at least two in your system; one allocated to the host system for system functions and the other available to isolate for use by a VM or application. 
+Isolating the GPU prevents apps and the system from accessing it.
+
```
Comment (clairewrtr) [inline content/SCALE/SCALEUIReference/SystemSettings/AdvancedSettingsScreen.md:208]: Line 208: Need to add a closing Expand tag.

**Review by clairewrtr — DISMISSED** [review]
Minor corrections, looks really good!

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**